Sources/AST/ASTPass/ASTPassContext.swift

```diff
@@ -150,6 +150,12 @@ extension ASTPassContext {
     set { self[IsExternalCallContextEntry.self] = newValue }
   }
 
+  /// The external call context while visiting an external call
+  public var externalCallContext: ExternalCall? {
+    get { return self[ExternalCallContext.self] }
+    set { self[ExternalCallContext.self] = newValue }
+  }
+  
```

Comment:
(The whitespace here causes the CI to fail.)

Reply:
Yes, just saw this. (sorry)